packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -100,9 +100,8 @@ abstract class FlutterCommand extends Command {
   // This is a field so that you can modify the value for testing.
   Validator projectRootValidator = () {
     if (!FileSystemEntity.isFileSync('pubspec.yaml')) {
-      printError('Error: No pubspec.yaml file found.\n'
-        'This command should be run from the root of your Flutter project.\n'
-        'Do not run this command from the root of your git clone of Flutter.');
+      printError('Error: No pubspec.yaml file found. This command should be run '
```

Comment:
Maybe we should autodetect that you're at the root of the flutter repo and only elaborate then? Or reword it a bit so it doesn't imply that you are at the root of the flutter repo?